tests/search/apps/mock_desktop_entries/falseapp.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=FalseApp
+X-GNOME-FullName=FalseApp - Full Name
+Comment=App 2 comment
+Exec=false
```

Comment:
`true` and `false` were picked because Exec needs to actually exist as a command, and they're harmless commands that does nothing, and that everyone has.